src/Bundle/Maker/MakeFactory.php

```diff
@@ -80,18 +90,19 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
     {
         $command
             ->setDescription(self::getCommandDescription())
-            ->addArgument('entity', InputArgument::OPTIONAL, 'Entity class to create a factory for')
+            ->addArgument('class', InputArgument::OPTIONAL, 'Entity, Document or class to create a factory for')
```

Comment:
this seemed more accurate to name this argument `class`

I don't think this introduce BC problem